asciidoc/modules/con_mg_use-future-methods-for-asynchronous-operations.adoc

```diff
@@ -33,9 +35,10 @@ response.onComplete(ar -> {
 });
 ----
 
-In basic applications, you will not notice distinct difference between using callbacks and futures. However, futures are very useful for composing asynchronous operations. When you use futures, the error handling is better. In callbacks, you have to handle failures at every stage of the composition, whereas in futures you can handle the failure once in the end.
+When you use futures, the error handling is better. In callbacks, you have to handle failures at every stage of the composition, whereas in futures you can handle the failure once in the end. In basic applications, you may not notice distinct difference between using callbacks and futures.
```

Comment:
how about "Error handling is better with futures" ?

Reply:
Done